pkg/ddc/base/runtime_helper.go

```diff
@@ -36,10 +37,16 @@ var (
 	// mem, ssd, hdd for Alluxio and GooseFS
 	// cache-dir for JuiceFS
 	cacheDirNames = []string{"datavolume-", "volume-localtime", "cache-dir", "mem", "ssd", "hdd"}
+
+	// hostpath fuse mount point for Alluxio, JindoFS, GooseFS and JuiceFS
+	hostMountNames = []string{"alluxio-fuse-mount", "jindofs-fuse-mount", "goosefs-fuse-mount", "juicefs-fuse-mount"}
+
+	// fuse devices for Alluxio, JindoFS, GooseFS
+	hostFuseDeviceNames = []string{"alluxio-fuse-device", "jindofs-fuse-device", "goosefs-fuse-device"}
 )
 
 // GetTemplateToInjectForFuse gets template for fuse injection
-func (info *RuntimeInfo) GetTemplateToInjectForFuse(pvcName string, enableCacheDir bool) (template *common.FuseInjectionTemplate, err error) {
+func (info *RuntimeInfo) GetTemplateToInjectForFuse(pvcName string, enableCacheDir bool, enableVirtFuseDev bool) (template *common.FuseInjectionTemplate, err error) {
```

Comment:
Could you use `option` struct instead of `enableCacheDir bool, enableVirtFuseDev bool`.

Reply:
Done. mv to `common.FuseSidecarInjectOptions`